src/server/generic_family_test.cc

```diff
@@ -453,14 +452,33 @@ TEST_F(GenericFamilyTest, Rename) {
   ren_fb.Join();
 }
 
-TEST_F(GenericFamilyTest, RenameNonString) {
-  EXPECT_EQ(1, CheckedInt({"lpush", "x", "elem"}));
-  auto resp = Run({"rename", "x", "b"});
-  ASSERT_EQ(resp, "OK");
-  ASSERT_EQ(2, last_cmd_dbg_info_.shards_count);
+TEST_F(GenericFamilyTest, RenameList) {
+  for (string_view dest : {"b", "y", "z"}) {
```

Comment:
running the same test for differrent destination keys - "b" on differrent shard; "y", "z" on the same shard.

checking that the memory usage is tracked as expected.